include/eve/module/core/regular/modf.hpp

```diff
@@ -7,10 +7,29 @@
 //==================================================================================================
 #pragma once
 
-#include <eve/detail/overload.hpp>
+#include <eve/arch.hpp>
+#include <eve/concept/value.hpp>
+#include <eve/traits/overload.hpp>
+#include <eve/module/core/decorator/core.hpp>
+#include <eve/module/core/regular/trunc.hpp>
+#include <eve/module/core/constant/zero.hpp>
+#include <eve/module/core/regular/zip.hpp>
+#include <eve/arch/platform.hpp>
 
 namespace eve
 {
+  template<typename Options>
+  struct modf_t : elementwise_callable<modf_t, Options, pedantic_option>
+  {
+    template<eve::value T>
+    EVE_FORCEINLINE constexpr eve::result_t<zip,T,T> operator()(T a) const noexcept
```

Comment:
let's have a typedef for `eve::result_t<zip,T,T>`
`zipped`?

Reply:
Maybe an alias `zipped<T..>` cause there is a bunch of variants.

Reply:
That's what meant, yes